themes/apachecon/layouts/tracks/section.html

```diff
@@ -1,47 +0,0 @@
-<!DOCTYPE HTML>
```

Comment:
We should not delete this file. 
Can you create a new PR just add upper two files?

Reply:
OK!